### PR TITLE
Update README.md (fix example), Add Python 3.6 support

### PR DESCRIPTION
**Pull request by stasfilin**
Hi, I fixed example from README.md and I tested fbtftp via Python 3.6, it's work well
- Add this example to examples dir
- Add Python 3.6 support

### CODE REVIEW

**Comment by stasfilin** [comment]
Update, i don't know why travis show error, but local:
```bash
Name                     Stmts   Miss  Cover
--------------------------------------------
fbtftp.py                    3      0   100%
fbtftp/base_handler.py     232     11    95%
fbtftp/base_server.py      125      3    98%
fbtftp/constants.py         20      0   100%
fbtftp/netascii.py          36      0   100%
--------------------------------------------
TOTAL                      416     14    97%
----------------------------------------------------------------------
Ran 35 tests in 2.103s

OK (SKIP=1)
python3 setup.py flake8
running flake8
(python36) [stas@archlinux fbtftp]$ python --version
Python 3.6.0
```

File: examples/baseserver.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
+from fbtftp.base_handler import BaseHandler
```
Comment (pallotron) [inline examples/baseserver.py]: isn't this file redundant? there is already `examples/server.py`

File: README.md [hunk 2]
```diff
@@ -179,10 +184,12 @@ class StaticServer(BaseServer):
             server_addr, peer, path, options, self._root,
             self._handler_stats_callback)
 
+
 def main():
-    server = StaticServer(ip='', port='1069', retries=3, timeout=5,
-                          root='/var/tftproot', print_session_stats,
-                          print_server_stats)
+    server = StaticServer(address='127.0.0.1', port=1069, retries=3,
```
Comment (pallotron) [inline README.md]: replace with `'::'`, in Facebook we consider IPv6 first class citizen.

Comment (pallotron) [inline README.md]: Also, I am failing to see what is the "fix" here... can you clarify?

**Comment by pallotron** [comment]
Also Travis CI is failing, here is the stacktrace from the build:

```
======================================================================
FAIL: testDownloadBulkSizes (integration_test.integrationTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/facebook/fbtftp/tests/integration_test.py", line 138, in testDownloadBulkSizes
    self.assertEqual(0, p_returncode)
AssertionError: 0 != 1
    '0 != 1' = '%s != %s' % _common_shorten_repr(0, 1)
    '0 != 1' = self._formatMessage('0 != 1', '0 != 1')
>>  raise self.failureException('0 != 1')
    
-------------------- >> begin captured logging << --------------------
root: INFO: Created temporary directory <TemporaryDirectory '/tmp/tmp56l5_noo'>
--------------------- >> end captured logging << ---------------------
```

Something up with `testDownloadBulkSizes` in py3.6

**Comment by pallotron** [comment]
ping.

**Comment by pallotron** [comment]
@stasfilin did you have a chance to look at the failed test?

**Comment by pallotron** [comment]
I know what's the problem with the broken test. I will fix in a separate commit.

**Comment by pallotron** [comment]
In tests/integration_test.py:

```patch
@@ -122,7 +122,7 @@
                 0,  # let the kernel decide the port
                 2,
                 2,
-                self.tmpdirname,
+                self.tmpdirname.name,
                 self.stats
             )
             child_pid = os.fork()
```

**Comment by pallotron** [comment]
doing this in https://github.com/facebook/fbtftp/pull/21

**Pull request closed without merging**